### ISSUE TITLE
Removed?

### ISSUE THREAD

**Issue by Tiny-Peanuts** [adamk33n3r/runelite-watchdog#89]
Did this plugin get removed? Doesn't seem to be on runelite anymore.

**Comment by adamk33n3r** [comment]
Yes it was removed by request from Jagex. I can reinstate it by removing the sound fired alerts. It's waiting on PR. https://github.com/runelite/plugin-hub/pull/4744
